mleap-spark-extension/src/main/scala/org/apache/spark/ml/mleap/classification/OneVsRest.scala

```diff
@@ -23,6 +23,7 @@ import java.util.UUID
 import ml.combust.mleap.core.annotation.SparkCode
 
 import scala.collection.JavaConverters._
+import scala.collection.parallel.CollectionConverters._
```

Comment:
@db-scnakandala There is an error while importing has this has been moved in Scala 2.13 to separate module [scala/scala-parallel-collection]
Need to update the library dependency such as for scala 2.13 onwards.
libraryDependencies += "org.scala-lang.modules" %% "scala-parallel-collections" % "1.0.0"

This is where one build is failing. 
`[error] /home/travis/build/combust/mleap/mleap-spark-extension/src/main/scala/org/apache/spark/ml/mleap/classification/OneVsRest.scala:26:34: object CollectionConverters is not a member of package scala.collection.parallel
[error] import scala.collection.parallel.CollectionConverters._`